### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by rymurr**

### CODE REVIEW

File: app/ui/Home.py [hunk 1]
```diff
@@ -1,46 +1,206 @@
...
+        return fmt(None)
+    return fmt(util)
+
+
+def to_str(end_time, start_time):
+    if not end_time:
+        return None, None
+    if not start_time:
+        return None, None
+    print(end_time, start_time, end_time.tzinfo, type(end_time.tzinfo))
```
Comment (joshelser) [inline app/ui/Home.py:50]:
```suggestion
```

File: app/ui/Home.py [hunk 2]
```diff
@@ -1,46 +1,206 @@
...
+        return fmt(None)
+    if df.empty:
+        return fmt(None)
+    util = df.iloc[0, 0]
+    if util is None:
+        return fmt(None)
+    return fmt(util)
+
+
+def to_str(end_time, start_time):
```
Comment (joshelser) [inline app/ui/Home.py:45]: Please add pydoc for this method since it returns two values.

File: app/ui/Home.py [hunk 3]
```diff
@@ -1,46 +1,206 @@
...
+
+    wh = get_wh_utilization()
+    st.metric("Warehouse Utilization", wh)
+    if wh == "Calculating":
+        st.markdown(
             """
+Note: Upon installation, the Sundeck native app analyzes your Snowflake usage
+history. Depending on the size of this data, this may takes minutes to hours.
+Once complete, utilization will reported above. (this is only shown until
+first materialization
```
Comment (joshelser) [inline app/ui/Home.py:132]: Missing end of this sentence?

> until [the] first materialization is complete)

File: app/ui/Home.py [hunk 4]
```diff
@@ -1,46 +1,206 @@
...
+The Sundeck native app automatically generates a fully materialized and incrementally
+updated set of warehouse and query activity data tables. This allows you to analyze
+key `SNOWFLAKE.ACCOUNT_USAGE` tables efficiently. These tables also are enriched to
+enhance analysis opportunities. Key tables include:
+
+#### REPORTING.ENRICHED_QUERY_HISTORY
+An enriched variation of `SNOWFLAKE.ACCOUNT_USAGE.QUERY_HISTORY`. Includes all
+columns from that tables as well as per query estimated costs using Sundeck's cost
+attribution model and automatically extracted QTag values. Additional details can be
+found here.
```
Comment (joshelser) [inline app/ui/Home.py:148]: > can be found here

Should the "here" words in this paragraph and below be hyperlinks to Sundeck docs?

File: app/ui/Home.py [hunk 5]
```diff
@@ -1,46 +1,206 @@
...
+    update_str = end_time.strftime("%Y-%m-%d %H:%M")
+    duration = timedelta_to_human_readable(end_time - start_time)
+    next_time_str = "60 minutes"
+
+    if duration is not None:
+        return f"{update_str} ({duration})", next_time_str
+    return update_str, next_time_str
+
+
+def get_refresh_data():
```
Comment (joshelser) [inline app/ui/Home.py:60]: pydoc to describe the values being returned would be helpful, too.

File: app/ui/Home.py [hunk 6]
```diff
@@ -1,46 +1,206 @@
...
+        else:
+            st.write(f"Data Available: {das} - {dae}")
+            st.write(f"Query History Last Update: {qlu}")
+            st.write(f"Query History Update Frequency: {qnu}")
+            st.write(f"Warehouse Events Last Update: {wlu}")
+            st.write(f"Warehouse Events History Update Frequency: {wnu}")
+
+    with st.expander("", expanded=True):
+        st.markdown(
+            """The native app is better when connecting to the Sundeck UI. This is
```
Comment (joshelser) [inline app/ui/Home.py:201]:
```suggestion
            """The native app is better when connected to the Sundeck UI. This is
```

File: app/ui/Home.py (hunk 2 above)
Comment (rymurr) [reply app/ui/Home.py:45]: done

File: app/ui/Home.py (hunk 1 above)
Comment (rymurr) [reply app/ui/Home.py:50]: done

File: app/ui/Home.py (hunk 3 above)
Comment (rymurr) [reply app/ui/Home.py:132]: fixed

File: app/ui/Home.py (hunk 5 above)
Comment (rymurr) [reply app/ui/Home.py:60]: done

File: app/ui/Home.py (hunk 6 above)
Comment (rymurr) [reply app/ui/Home.py:201]: fixed

File: app/ui/Home.py (hunk 4 above)
Comment (rymurr) [reply app/ui/Home.py:148]: added a few, waiting on feedback from J for the rest

**Comment by jacques-n** [comment]
Let's do a few updates:

* For the boxes on the right, let's add small titles to each (h3 maybe?)
   * Box1: Current Data Status
   * Box2: Connect to Sundeck
* For warehouse utilization metric, maybe put "Your current aggregate warehouse utilization:" instead of "Warehouse utilization"
* Let's update the logo to span the entire page (put right hand boxes below it). Let's also shrink it by 25%.
* For non-obvious links, just remove that portion of the sentence for now.
* Let's invert the management tools and the consumption table sections so management tools come first.

**Comment by rymurr** [comment]
2nd iteration

**Review by joshelser — DISMISSED** [review]
lgtm

**Review by joshelser — DISMISSED** [review]

**Pull request merged**